tox.ini

```diff
@@ -80,6 +81,22 @@ commands =
     python -c 'from shutil import rmtree; rmtree("build", True); rmtree("dist", True)'
     python -m build .
 
+[syftcli.build]
+basepython = python3
+changedir = {toxinidir}/packages/syftcli
+description = Build SyftCLI Binary for each platform
+commands =
+    python -m pip install --upgrade pip
+    pip install --upgrade setuptools wheel twine tox build
```

Comment:
NITPICK: We could remove this pip statement, as we would probably not need these pacakges for pyinstaller building?